### PR TITLE
[DOCFIX] Update web interface doc

### PR DESCRIPTION
**Pull request by tieujason330**
Made some minor tweaks - typos

### CODE REVIEW

**Comment by tieujason330** [comment]
jenkins, test this please

**Comment by tieujason330** [comment]
@ZacBlanco PTAL, thanks

File: docs/en/basic/Web-Interface.md [hunk 1]
```diff
@@ -118,7 +118,7 @@ but shows information specific to the particular worker.
 ### BlockInfo Page
 
 In the "BlockInfo" page, information on the files stored by the worker is shown,
-such as the file size and which tiers the file is stored on.
+such as the file size and which tier the file is stored on.
```
Comment (ZacBlanco) [inline docs/en/basic/Web-Interface.md]: It's technically possible for blocks to be stored in multiple tiers across different workers. Will the UI show both?

Comment (tieujason330) [reply docs/en/basic/Web-Interface.md]: I'm not sure, is there a way I can manually store blocks on multiple tiers to test?

The UI code only shows 'Tier', so I assumed it was single.

Comment (ZacBlanco) [reply docs/en/basic/Web-Interface.md]: If the UI only specifies "Tier" for now, then I think we can leave it at that.

**Comment by cheng-chang** [comment]
alluxio-bot, merge this please

**Comment by calvinjia** [comment]
alluxio-bot, cherry-pick this to branch-2.0 please.

**Pull request merged**